src/main/java/htsjdk/samtools/SAMRecord.java

```diff
@@ -1154,12 +1168,35 @@ public Object getAttribute(final String tag) {
         return getAttribute(SAMTag.makeBinaryTag(tag));
     }
 
+    /**
+     * Get the value for a SAM tag.
+     * WARNING: Some value types (e.g. byte[]) are mutable.  It is dangerous to change one of these values in
+     * place, because some SAMRecord implementations keep track of when attributes have been changed.  If you
+     * want to change an attribute value, call setAttribute() to replace the value.
+     *
+     * @param tag Two-character tag name.
```

Comment:
Since it's no longer a string:
```suggestion
     * @param tag the SAM tag
```

Reply:
👍